src/main/java/de/chojo/repbot/data/GuildData.java

```diff
@@ -376,4 +376,16 @@ ON CONFLICT(guild_id)
                 .readRow(rs -> rs.getLong("user_id"))
                 .allSync();
     }
+
+    public void isChannelWhitelist(Guild guild, boolean whitelist) {
```

Comment:
Should probably be renamed to something with set as this method doesn't query any information